AndelaOSP/Andela-Socials
# #166153617 (feat): Add event to creators calendar

#### What Does This PR Do?
- adds a method call to add an event to the creator's calendar 

#### Description Of Task To Be Completed
- As an event creator, when I create an event, the event should automatically be added to my calendar after creation.

#### Any Background Context You Want To Provide?
- N/A

#### How can this be manually tested?
- clone or fetch from this repo and switch to the branch `ft-add-event-to-creators-calendar-166153617`
- create an event using the frontend client
- check your calendar for the added event.

#### What are the relevant pivotal tracker stories?
[#166153617 As an event creator, an event that I create should be added to my calendar after creation](https://www.pivotaltracker.com/story/show/166153617)

#### Screenshot(s)
![image](https://user-images.githubusercontent.com/28973383/58148128-245b9180-7c66-11e9-9709-2823a1f6b5ef.png)
![image](https://user-images.githubusercontent.com/28973383/58148166-4fde7c00-7c66-11e9-94f1-39bccccb56fd.png)
